test/e2e/features_test.go

```diff
@@ -135,6 +135,18 @@ func TestWebhookUpdate(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// on KIND, this test requires webhookKINDConfig. the generic getEnvironmentBuilder we use for most tests doesn't
+	// support this: the configuration is specific to KIND but should not be used by default, and the scaffolding isn't
+	// flexible enough to support tests building their own clusters or passing additional builder functions. this still
+	// uses the setup style from before getEnvironmentBuilder/GKE support as such, and just skips if it's attempting
+	// to run on GKE
```

Comment:
I modified the [issue](https://github.com/Kong/kubernetes-ingress-controller/issues/1613) to have this tracked.